src/components/Calculator.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import calculate from './logic/calculate';
```

Comment:
- [ ] Kindly remove the import statement for the calculate component, doing this will clear the error seen in your page 👍🏿